crates/fuel-core/src/database/message.rs

```diff
@@ -27,19 +37,50 @@ use std::{
     ops::Deref,
 };
 
-use super::storage::DatabaseColumn;
+fuel_core_types::fuel_vm::double_key!(OwnedMessageKey, Address, address, Nonce, nonce);
+
+/// The table that stores all messages per owner.
+pub struct OwnedMessageIds;
+
+impl Mappable for OwnedMessageIds {
+    type Key = OwnedMessageKey;
+    type OwnedKey = Self::Key;
+    type Value = ();
```

Comment:
The type of value written to the database is `()` - Is this right?

Reply:
Added additional test to prove all possible use cases